core/src/main/scala/org/apache/spark/ui/jobs/AllStagesPage.scala

```diff
@@ -34,9 +34,9 @@ private[ui] class AllStagesPage(parent: StagesTab) extends WebUIPage("") {
     listener.synchronized {
       val activeStages = listener.activeStages.values.toSeq
       val pendingStages = listener.pendingStages.values.toSeq
-      val completedStages = listener.completedStages.reverse.toSeq
+      val completedStages = listener.completedStages.reverse
```

Comment:
If it's really matter, it's better to add type to `val completedStages`, it describes your point much better then trying call no-op method. But it won't be consistent, so I would prefer to leave it as is.